errors/index.js

```diff
@@ -13,7 +13,7 @@ exports.handlePsqlErrors = (err, req, res, next) => {
 };
 
 exports.handleServerErrors = (err, req, res, next) => {
-   
+   console.log(err)
```

Comment:
Nice extraction of logic to errors file! Definitely keep this up with your other error handling middleware 💯